covfee/client/journey/journey.tsx

```diff
@@ -330,6 +344,17 @@ const ContentContainer = styled.div<any>`
   overflow: auto;
 `
 
+const ContentContainerWithoutSidebar = styled.div<any>`
+  position: fixed;
+  top: 46px;
+  right: 0;
+  display: inline-block;
+  vertical-align: top;
+  height: calc(100vh - 46px);
+  width: calc(100%);
```

Comment:
Instead of duplicating the code of ContentContainer, you can make it configurable

```
interface ContentContainerProps {
  showSideBar: boolean
}

const ContentContainer = styled.div<ContentContainerProps>`
  position: fixed;
  top: 46px;
  right: 0;
  display: inline-block;
  vertical-align: top;
  height: calc(100vh - 46px);
  width: ${(props) => (props.showSideBar ? "calc(100% - 25%)" : "100%")};
  overflow: auto;
`
```

then instantiate, with `<ContentContainer showSideBar={args.showSideBar}>`